lib/ddr/models/language.rb

```diff
@@ -0,0 +1,31 @@
+require "active_resource"
+
+module Ddr::Models
+  class Language < ActiveResource::Base
+
+    self.site = ENV["DDR_AUX_API_URL"]
+
+    def self.call(obj)
```

Comment:
It doesn't right now. Eventually I'd like the indexing behavior to enforce using the codes and ignore or raise an error when a value in language isn't a valid code, but we're not there yet. I'd considered making the behavior of self.call be equivalent to what's currently happening in `Ddr::Models::Indexing#language_name` but it seemed odd to return a mix of strings and Language instances from self.call. Happy to reconsider.